process/process_linux_test.go

```diff
@@ -59,3 +60,30 @@ func BenchmarkLinuxAllProcessesOnLocalProcFS(b *testing.B) {
 		b.Fatal("non-zero errors for AllProcesses", errCount)
 	}
 }
+
+func TestFillNSPidFromStatus(t *testing.T) {
```

Comment:
Lets add a test case for kernel versions that don't have `NSPid`

Reply:
Thanks for the suggestion! I'll run the same test program on an older Linux version, collect the `procfs` files and then write unit test to check the expected behavior!